include/zarray/zarray_impl.hpp

```diff
@@ -430,6 +435,14 @@ namespace xt
         return new self_type(*this);
     }
 
+    template <class CTE>
+    inline zarray_impl* zexpression_wrapper<CTE>::make_view(xstrided_slice_vector& slices)
+    {
+        auto e = strided_view(m_expression, slices);
+        zarray_impl* p_impl = e.derived_cast().allocate_result();
```

Comment:
`e.derived_cast()` is an xexpression that do not provide the `allocate_result` method. What you want to do here is build a `zexpression_wrapper` on the newly created strided_view:

```cpp
return build_zarray(std::move(e));
```